dev/Dockerfile

```diff
@@ -62,7 +60,7 @@ RUN chmod u+x /opt/spark/sbin/* && \
 
 RUN pip3 install -q ipython
 
-RUN pip3 install "pyiceberg[s3fs]==${PYICEBERG_VERSION}"
+RUN pip3 install "pyiceberg[s3fs,pyarrow,hive]==${PYICEBERG_VERSION}"
```

Comment:
Do we need `pyarrow` in the docker image? I think in `provision.py` we only need pyiceberg to create some empty table with UUID column. Since we use minio for storage, `s3fs` can help us write the metadata

Reply:
Thanks! I got an error before, and therefore I added it, but I've removed it again and it seems to work fine 👍